test/stubs/systemInfo_stub.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * systemInfo_stub.c
+ *Stub for the function get_current_time
+ *
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+int* time_to_return;
+int current_call = 0;
+
+void set_current_time(int* times){
+  time_to_return = (int*) malloc(sizeof(times));
```

Comment:
I don't love that we are using `malloc` and `memcpy` here... it seems error prone and sets a bad precedent for the use of dynamic memory in the repository. Can you find an alternative? 

I know this is test code, so its not including any dynamic memory allocation in the release build, but we should just avoid it entirely if we can.

One option might be to have a module variable predefined with a limit and just call `set_time(int x)` several times from the test to fill up this module variable. Might also require an `init_time()` function.